app/src/main/java/com/cornellappdev/uplift/MainActivity.kt

```diff
@@ -14,10 +15,9 @@ class MainActivity : ComponentActivity() {
         val gymDetailViewModel = GymDetailViewModel()
 
         gymDetailViewModel.selectGym(testMorrison)
-
         setContent {
             UpliftTheme {
-                MainNavigationWrapper(gymDetailViewModel = gymDetailViewModel)
+               MainNavigationWrapper(gymDetailViewModel = gymDetailViewModel)
             }
```

Comment:
Please run a linter on the codebase before every commit! You can quickly do so by right clicking on our origin package:

![image](https://user-images.githubusercontent.com/47724806/227789850-4501329a-5f90-42d6-b7c7-c3f7b7cca553.png)